code/cutscene/cutscenes.h

```diff
@@ -13,12 +13,16 @@
 #include "globalincs/globals.h"
 #include "globalincs/pstypes.h"
 
+#define CF_VIEWABLE			(1<<0)
+#define CF_ALWAYS_VIEWABLE	(1<<1)
+#define CF_NEVER_VIEWABLE	(1<<2)
+
 typedef struct cutscene_info
 {
 	char filename[MAX_FILENAME_LEN];
 	char name[NAME_LENGTH];
 	char* description;
-	bool viewable;
+	int flags;
```

Comment:
Please use a `uint` or even better a `uint32_t` for bitfields.

Reply:
Ok.